src/redis-check-rdb.c

```diff
@@ -272,6 +272,21 @@ int redis_check_rdb(char *rdbfilename, FILE *fp) {
             decrRefCount(auxkey);
             decrRefCount(auxval);
             continue; /* Read type again. */
+        } else if (type == RDB_OPCODE_MODULE_AUX) {
+            /* AUX: Auxiliary filed data for modules.
+             */
+            uint64_t moduleid, when_opcode, when;
+            rdbstate.doing = RDB_CHECK_DOING_READ_AUX;
+            if ((moduleid = rdbLoadLen(&rdb,NULL)) == RDB_LENERR) goto eoferr;
+            if ((when_opcode = rdbLoadLen(&rdb,NULL)) == RDB_LENERR) goto eoferr;
+            if ((when = rdbLoadLen(&rdb,NULL)) == RDB_LENERR) goto eoferr;
+
+            char name[10];
+            moduleTypeNameByID(name,moduleid);
+            rdbCheckInfo("Type name %s", name);
```

Comment:
i'm not sure we need this.
it exists in RDB_OPCODE_AUX because it prints the version of the redis that generated the rdb and other global info. not sure it is needed here too.

on the other hand, rdb files with this info will be rare, and even when found, we'll only have maximum of 2 prints per module per rdb, so not flooding the log file.

Reply:
if we don't drop this, let's at least change the "Type name" to be more descriptive.
something like "MODULE AUX for: %s"
?